source/_integrations/homewizard.markdown

```diff
@@ -23,8 +23,15 @@ Integration for the [HomeWizard Energy](https://www.homewizard.nl/energy) platfo
 
 - [Wifi P1 Meter](https://www.homewizard.nl/p1-meter): Depending on the connected DSMR meter: sensors for power import/export, energy consumption (single or three phases) and gas. (Model: `HWE-P1`)
 - [Wifi Energy Socket](https://www.homewizard.nl/energy-socket): Sensors for power import/export and energy consumption and switches for controlling the outlet (model: `HWE-SKT`)
+- [Wifi Watermeter](https://www.homewizard.com/watermeter): Sensors for active and total water usage (model: `HWE-WTR`)
 - [Wifi kWh Meter](https://www.homewizard.nl/kwh-meter): Sensors for power import/export and energy consumption. (Models: `SDM230-wifi`, `SDM630-wifi`)
 
+<div class='note'>
+
+The Wifi Watermeter can be powered via a USB-C cable and with batteries. When using batteries the watermeter only connects to Wifi every couple of hours. Because of this the API can only be used when powered via the USB-C cable. It is not possible to add use this integration when the watermeter is powered with batteries. 
```

Comment:
```suggestion
The Wi-Fi Watermeter can be powered via a USB-C cable and with batteries. When using batteries they only connect to Wi-Fi every couple of hours. Because of this, the API can only be used when powered via the USB-C cable. It is not possible to add use this integration when the water meter is powered by batteries.
```

Reply:
Let me change all the ‘Wifi’ to ‘Wi-Fi’, as that is also how the devices are named by HomeWizard.